### ISSUE TITLE
Include d3-geo-polygon too (optional?)

### ISSUE THREAD

**Issue by Fil** [vega/vega-projection-extended#1]
See https://beta.observablehq.com/@fil/vega-projections and https://github.com/vega/vega/issues/1292

(The more projections, the merrier the cartographer.)

**Comment by mattijn** [comment]
Yes, I'm also interested in the Dymaxion projection, which is included in the `d3-geo-polygon` package as `airocean`.
